content/windows/docs/hardening.md

```diff
@@ -1,37 +1,20 @@
 # Hardening the Windows worker node
 
-Windows Server hardening involves identifying and remediating security vulnerabilities before they are exploited. 
+Windows Server hardening involves identifying and remediating security vulnerabilities before they are exploited. Amazon EKS Optimized Windows AMIs are not hardened by default and only contain the latest Microsoft security patches and EKS components. 
```

Comment:
The second sentence seems to contradict itself. They're not hardened but they container the latest security patches? Can you re-work this @marciogmorales?

Reply:
I think a brief explanation of "hardening" in this context would help.